clc/modules/block-storage/src/main/java/com/eucalyptus/blockstorage/DASManager.java

```diff
@@ -787,6 +787,46 @@ public boolean shouldExportOnReload(LVMVolumeInfo lvmVolumeInfo) {
       return returnValues;
     }
 
+    protected String enableLogicalVolume(String lvName) throws EucalyptusCloudException {
+      
+      EucalyptusCloudException ex = null;
+      String result = null;
+
+      // 1st timeout is 10 ms, 2nd is 40. Last (9th) is 2.56 sec. Total 3.41 sec.
+      int attempt = 1;
+      final int max_attempts = 9;
```

Comment:
Just curious why the number of attempts is 9, why not 10 or a configurable value